src/scss/src/foundation/_variables.scss

```diff
@@ -258,6 +258,42 @@ $tt: (
 =                BREAKPOINTS                  =
 =============================================*/
 
-/* Breakpoints */
+$mobile: 25.875rem; // 414px
+$tablet: 55rem; // 880px
+$desktop: 90rem; // 1440px
+
+$breakpoints: (
+  mobile: $mobile,
+  tablet: $tablet,
+  desktop: $desktop
+);
+
+/*=============================================
+=                  SPACING                   =
+=============================================*/
+
+$spacing-none: 0;
+$spacing-xxxs: 0.25rem; // 4px
+$spacing-xxs: 0.5rem; // 8px
```

Comment:
Don't these spacing keys like `xxxs`, `xxs` and so on look a bit unusual in comparison to other design systems?
Wouldn't it be more convenient to use for spacing something like `spacing-1`, `spacing-2` similar to the [Carbon design system](https://carbondesignsystem.com/elements/spacing/overview/#spacing-scale), for example?

Reply:
I think T-shirt-like approach will be more suitable because I hope we will create `Margin` and `Padding` or `Spacing` components like following (code from senior react course: https://softserveacademy.udemy.com/course/react-for-senior-engineers/learn/lecture/26838048#questions)

Example of `Margin` component: 
```jsx
import React from 'react'
import { Spacing } from '@s2s/foundation'

export interface MarginProps {
    space?: keyof typeof Spacing
    left?: boolean,
    right?: boolean,
    top?: boolean,
    bottom?: boolean
}

const Margin: React.FC<MarginProps> = ({ space = 'xxxs', children, left, right, top, bottom }) => {
    let className = ``

    if (!left && !right && !top && !bottom) {
        className = `s2s-margin-${space}`
    }

    if (left) {
        className = `${className} s2s-margin-left-${space}`
    }

    if (right) {
        className = `${className} s2s-margin-right-${space}`
    }

    if (top) {
        className = `${className} s2s-margin-top-${space}`
    }

    if (bottom) {
        className = `${className} s2s-margin-bottom-${space}`
    }

    return <div className={className}>
        {children}
    </div>
}

export default Margin
```

And example of usage:

```jsx
<Margin left space="lg">
     <div>Content with large left margin</div>
</Margin>

<Margin right space="xs">
      <div>Content with extra small right margin</div>
</Margin>
```

As you can see we will pass `space` prop, which will be autocompleting sizes of spacing like `xxxs`, `xxs`, and so on. In my opinion `spacing-1`, `spacing-2` options and so on is not very readable when using as a prop

Reply:
From this point of view, it makes sense :)